yara/style.css

```diff
@@ -0,0 +1,173 @@
+*
+{
+    margin: 0;
+    padding: 0;
+}
+.logo
+{
+    height: 80px;
+    width: 80px;
+    margin-left: 20px;
+}
+
+.header
+{
+    background-color: thistle;
+    box-shadow: 2px 5px 8px #888888;
+    top: 0;
+    width: 100%;
+    position: fixed;
+    z-index: 1;
```

Comment:
I'm not sure you need to use z-index here, it will be over the other views since it has a fixed position already (: